nextflow/src/org/labkey/nextflow/pipeline/NextFlowRunTask.java

```diff
@@ -54,9 +49,9 @@ public NextFlowRunTask(Factory factory, PipelineJob job)
 
         // NextFlow requires a unique job name for every execution. Increment a counter to append as a suffix to
         // ensure uniqueness
-        long invocationCount = INVOCATION_SEQUENCE.next();
-        INVOCATION_SEQUENCE.sync();
-        NextFlowPipelineJob.LOG.info("Starting to execute NextFlow: {}", getJob().getJsonJobInfo(invocationCount));
+        NextFlowManager.get().incrementInvocationCount(getJob());
```

Comment:
should this be in try catch?

Reply:
I don't think that would add anything. We need to increment the number whether the execution succeeds or not. I need the new value for the logging call on the next line, so I can't really delay it either.